OpenImis.ePayment/Logic/PaymentLogic.cs

```diff
@@ -34,74 +35,18 @@ public PaymentLogic(IConfiguration configuration, IHostingEnvironment hostingEnv
         public async Task<DataMessage> SaveIntent(IntentOfPay intent, int? errorNumber = 0, string errorMessage = null)
         {
 
-            ImisPayment payment = new ImisPayment(_configuration, _hostingEnvironment);
-            var intentResponse = await payment.SaveIntentAsync(intent, errorNumber, errorMessage);
+            var cnHandler = new SingleControlNumberHandler(_configuration, _hostingEnvironment);
+            var return_message = cnHandler.GetControlNumber(intent);
```

Comment:
Does the previous version of requesting 1 CN from a Payment Switch still works? If that's not the case, we should verify if there is available CN for the product and send one (bulk CNs) OR create the intent and request 1 CN to GePG.